packages/salesforcedx-vscode-core/package.nls.ja.json

```diff
@@ -51,7 +51,7 @@
   "project_generate_with_manifest_text": "SFDX: マニフェストファイルを使用してプロジェクトを作成",
   "project_retrieve_start_default_org_text": "SFDX: デフォルトのスクラッチ組織からソースをプル",
   "project_retrieve_start_ignore_conflicts_default_org_text": "SFDX: Pull Source from Default Org and Ignore Conflicts",
-  "push_or_deploy_on_save_enabled_description": "ローカルのソースファイルを保存した際に、自動的に project:deploy:start コマンド (ソースが追跡される組織) または、force:source:deploy コマンド (ソースが追跡されない組織) を実行するかどうかを指定します。",
+  "push_or_deploy_on_save_enabled_description": "ローカルのソースファイルを保存した際に、自動的に project:deploy:start コマンド (ソースが追跡される組織) または、project:deploy:start コマンド (ソースが追跡されない組織) を実行するかどうかを指定します。",
```

Comment:
I saw that in the English you used the words `push` and `deploy` in this message rather than just saying `project:deploy:start` for both.  I like that wording change, since it's much more clear - can you please copy over the English wording to replace this Japanese one?  (We can get the translation later.)